### ISSUE TITLE
playground: create a simple tour for users onboarding

### ISSUE THREAD

**Issue by xmonader** [threefoldtech/tfgrid-sdk-ts#736]
https://tourguidejs.com/#guide

**Comment by MohamedElmdary** [comment]
Tried tourguidejs but they doesn't have a good ts support
so I tried https://introjs.com/docs/hints/api almost the same api with better ts support

can you provide some storied for guide like profile manager (select x then y and what text to show) ... etc ? @xmonader

**Comment by xmonader** [comment]
I don't mind any option that works well :)  Will help you on that, or even better @Mik-TF would you like to contribute to this?

**Comment by Mik-TF** [comment]
# Playground with Intro.JS Stories

* Page: Connect your TFChain Wallet (first visit)
  * 3 steps
    * Subpage Connect Your Wallet
      * step 1: info on mnemonics
        * can simply display the message that appears when you hover the box
      * step 2: info on password
        * idem as above
    * Subpage Login
      * step 3: info on login
        * Message: "Once you've set your mnemonics, you can login into the playground simply by entering the password associated with your mnemonics"

* Page: Connect your TFChain Wallet (after login)
  * 2 steps
    * Step 1: Info on how to send TFT to TFChain with QR code
    * Step 2: Info on how to update or add a Publich SSH key

For the rest, we could start by given steps to deploy a Full vm and micro VM. We can use the text in the info button as the steps (we can simplify the text).

* Page: Full VM before deployment
  * Steps for Config
    * Name: Text: "Choose a name for your instance or leave the auto-generated name. Note: only lower-case letters and numbers allowed."
    * choose vm image: Text: "you can choose a supported VM image or import a custom FList (more on info [link to manual])"
    * capacity: Text: "You can choose a preconfigured capacity settings or set your own custom capacity."
    * network: Text: "Select a network for your workload connection. 
    * GPU, dedicated, certified: Text: put text from info buttons
    * deploy: Text:  "Once you're ready, click deploy."
  * Steps for Disks
    * Text: "Click the + button to add a disk to your deployment. This is optional for full VM."

* Page: Full VM after deployment
  * Steps: explain the contract details page
    * Details vs JSON page
      * Text: "You can consult the Details or JSON format on the deployment details.
    * IP addresses
      * Text: "You can access your deployment's IP address (planetary, ipv4, ipv6, wireguard)"
    * Wireguard configuration
      * Text: "To set a wireguard configuration, copy this content in a .conf file. Read more [manual link]"
    * Metrics
      * Text: "copy this url at all time to a web browser to consult your deployment's status"


***

I'll wait for your input before going any further! Let me know what you guys think.

**Comment by MohamedElmdary** [comment]
Thanks for providing info mr @Mik-TF I'm still checking available apis and libs. for intro.js it works just fine but seems like there will be some issue with hidden css style for example setting a position of something using transition or a hidden content won't work that great that why we need some advanced usage from api

**Comment by MohamedElmdary** [comment]
Currently, I'm blocked on finding if a dialog is opened or not because it leads to unexpected issue
Idea to fix it:
Extends VDialog from vuetify and configure a global lock to await until there is no tour/dialog is opened

**Comment by MohamedElmdary** [comment]
Should be fixed after Creating a lock 
https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1294/files#diff-cb9af63a16e36c1e24d4dccece93742963ae7333f768d5202c9ffb5d41b7b540

And extend vuetify dialog to use the lock
https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1294/files#diff-3e2a0d53d65c312f65ff2e7a1fae8f24c0daeeb6e7bcd7fa5ac16338fd98be34

**Comment by MohamedElmdary** [comment]
- issue require a lot of code duplication which can't be avoided as it update some html in each solution
- we still need text to write in each solution

**Comment by ramezsaeed** [comment]
make sure to have skip button , so users can skip it if not needed.

**Comment by xmonader** [comment]
won't do
